src/sentry/api/endpoints/grouping_levels.py

```diff
@@ -94,9 +89,13 @@ def get_levels_overview(group):
             [
                 Column("primary_hash"),
                 Function(
-                    "max", [Function("length", [Column("hierarchical_hashes")])], "num_levels"
+                    "argMax",
+                    [
+                        Column("hierarchical_hashes"),
+                        Function("length", [Column("hierarchical_hashes")]),
```

Comment:
Instead of sending the postgres hashes to Snuba using `_current_level_expr`, we can simply return the longest array here and do the computation on the clientside.

This query takes up to 40ms in prod. I wonder if there's a faster way to compute what we're computing here, and also how to cache this with proper cache invalidation